unitTests/ui/FacetValueSuggestionsTest.ts

```diff
@@ -368,21 +372,47 @@ export function FacetValueSuggestionsTest() {
       });
     });
 
-    describe('when only using the search box keywords and the keyword is empty', () => {
+    describe('when only using the search box keywords', () => {
       beforeEach(() => {
         test.cmp.options.useValueFromSearchbox = true;
         test.cmp.options.useQuerySuggestions = false;
-        setUpKeywordInOmnibox('');
       });
 
-      it('returns no suggestions', async done => {
+      it(`returns no suggestions`, async done => {
+        setUpKeywordTextInOmnibox(anOmniboxSuggestionKeyword.text);
+        const resultingArgs = await triggerPopulateOmniboxEvent();
+
+        firstSuggestion(resultingArgs).then(result => {
+          expect(result.length).toBe(0);
+          done();
+        });
+      });
+
+      it('still returns no suggestions if the omnibox is empty', async done => {
+        setUpKeywordTextInOmnibox('');
         const resultingArgs = await triggerPopulateOmniboxEvent();
 
         firstSuggestion(resultingArgs).then(result => {
           expect(result.length).toBe(0);
           done();
         });
       });
+
+      it(`calls the provider with the omnibox's text if it isn't empty`, async done => {
```

Comment:
I think it would be clearer to mention `getSuggestions` in this and the next test's descriptions

Reply:
Good idea, I'm making those changes:
`calls the provider with the omnibox's text if it isn't empty` → `calls the provider's getSuggestions with the omnibox's text if it isn't empty`
`doesn't call the provider if the omnibox is empty` → `doesn't call the provider's getSuggestions if the omnibox is empty`